src/vireshJJun20/BankOps.java

```diff
@@ -0,0 +1,80 @@
+package vireshJJun20;
+
+public class BankOps {
+	static int availableBal=10000;
+	static int allCredit=0;
+	static int allDebit=0;
+	static int allPrintBal=0;
+	int creditCnt=0;
+	int	debitCnt= 0;
+	int printCnt=0;
+	
+	int creditTr(int cr) {
```

Comment:
I always told you will review first code, then why are you have not closed this PR and generated new, one file PR.

Reply:
After lots of discussion with me, it's not excepting from your side.